MacVM/VMInstance.swift

```diff
@@ -233,7 +233,15 @@ class VMInstance: NSObject, VZVirtualMachineDelegate {
         } catch {
             NSLog("Storage Error: \(error)")
         }
-        
+
+        let sound_config = VZVirtioSoundDeviceConfiguration()
+        let sink_config = VZVirtioSoundDeviceOutputStreamConfiguration()
+        sink_config.sink = VZHostAudioOutputStreamSink()
+        sound_config.streams.append(sink_config)
+        let source_config = VZVirtioSoundDeviceInputStreamConfiguration()
+        source_config.source = VZHostAudioInputStreamSource()
+        sound_config.streams.append(source_config)
```

Comment:
Thanks for submitting this PR 🙏

In this project we prefers to use camel case instead of snake case for variable names. Do you mind to update the variable names and then I can merge them?

Reply:
Sure thing!